metadata-doc.md

```diff
@@ -278,14 +342,15 @@ erDiagram
     string container  
   }
   indexes{
```

Comment:
index.yaml (here indexes) is a client-side concept to facilitate reducing the number of properties we have to input to import an image, not really something we'd expect to be in the API or database.

Reply:
Fixed in #11 -> https://github.com/ai-cfia/nachet-metadata-format/commit/81ff9b99a16d744d8afb4ee631128213ddb586da